exercises/phone-number/test/test_phone_number.c

```diff
@@ -2,15 +2,18 @@
 #include "../src/phone_number.h"
 #include <stdlib.h>
 
-char *result;
+char *result = NULL;
```

Comment:
How about making this guy `static`?

Reply:
Do we care that this would mask the compiler error for uninitilized variables?

Reply:
By this I meant the = NULL and not the static.

Reply:
So the addition of the `= NULL` actually does nothing, since globals are always zero initialized. I usually write it myself just to be clear. I don't *think* either way causes a compiler error though.

It does need to be static to match our conventions though, you're right @ryanplusplus